### PR TITLE
Adding PowerFlex in installation wizard ui

### PR DESCRIPTION
**Pull request by samihan-dell**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/698 |

# Checklist:

- [X] Have you run a grammar and spell checks against your submission?
- [X] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/deployment/csminstallationwizard/src/index.html [hunk 1]
```diff
@@ -331,6 +331,22 @@
...
 
+                  <div class="row mb-4 monitor">
+                    <div class="col-sm-12">
+                      <input class="form-check-input" type="checkbox" id="monitor" value="">
+                      <label for="monitor" class="form-check-label ms-2 text-dark" data-bs-toggle="tooltip" data-bs-placement="right" title="Allows the configuration of the SDC monitoring pod.">Allow Monitoring Pod</label>
+                    </div>
+                  </div>
+
+                  <div class="row mb-4 cert-secret-count-wrapper">
+                    <label for="cert-secret-count" class="col-sm-2 col-form-label" data-bs-toggle="tooltip" data-bs-placement="right" title="Represents the number of certificate secrets, which the user is going to create for SSL authentication." style="width: 180px;">CERT Secret Count</label>
```
Comment (niranjan-n1) [inline content/docs/deployment/csminstallationwizard/src/index.html:342]: Change CERT to Cert/Certificate

**Review by niranjan-n1 — CHANGES_REQUESTED** [review]
minor comment

File: content/docs/deployment/csminstallationwizard/src/index.html (hunk 1 above)
Comment (samihan-dell) [reply content/docs/deployment/csminstallationwizard/src/index.html]: Done

**Review by niranjan-n1 — APPROVED** [review]
LGTM

**Review by shanmydell — APPROVED** [review]

**Pull request merged**